docs/integrations-guides/rollup-guides/op-stack/README.md

```diff
@@ -22,7 +22,10 @@ First check out the version of the EigenDA proxy corresponding to the version of
 |------------------|-----------------------|
 | [v1.7.6](https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.6)           | [v1.0.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.0.0)                |
 | [v1.7.7](https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.7)           | [v1.2.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.2.0)                |
-| [v1.9.0](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.0)           | [v1.4.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.4.0)
+| [v1.9.0](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.0)           | [v1.4.0](https://github.com/Layr-Labs/eigenda-proxy/releases/tag/v1.4.0)                |
+| [v1.9.3](https://github.com/ethereum-optimism/optimism/releases/tag/v1.9.3)           | TODO(@bxue): make new release                |
```

Comment:
TODO for release captain @bxue-l2

Reply:
released 1.5.0

Reply:
[57cc70c](https://github.com/Layr-Labs/eigenda-docs/pull/20/commits/57cc70c87ffff43d238f3ee621cd12b17a3f7efd)